02-s3/README.md

```diff
@@ -121,21 +121,30 @@ Add an object to your bucket:
 
 _How would you copy the contents of the directory to the top level of your bucket?_
 
+> Answer: you can use aws `s3 localdir S3://bucket --recursive`
+
 ##### Question: Directory Copying
 
 _How would you copy the contents and include the directory name in the s3 object
 paths?_
 
+> Answer: I couldn't find a way to do this with cp but sync worked with `aws s3 sync .  s3://stelligent-u-matthew.holmes.labs` 
+> Although that everything in this module directory. I couldn't find a way to copy just one whole directory intact
+
 ##### Question: Object Access
 
 _[Can anyone else see your file yet](https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-access-control.html)?_
 
 For further reading, see the S3 [Access Policy Language Overview](https://docs.aws.amazon.com/AmazonS3/latest/dev/access-policy-language-overview.html).
 
+> Answer: By default s3 buckets and objects are private
+
 ##### Question: Sync vs Copy
 
 _What makes "sync" a better choice than "cp" for some S3 uploads?_
 
+> sync seemed better for working with multiple objects as well as updating existing objects which copy didn't do. 
+
```

Comment:
It also includes a `--delete` option that makes it easy to keep the destination clean.